hack/e2e-kind-cluster-setup.sh

```diff
@@ -25,18 +30,47 @@ setup_cluster() {
 push_local_image() {
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make manifests
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make img-build
-    kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+
+    if [[ $CLUSTER_TYPE == "kind" ]]; then
+      kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+    else
+      "$OCI_BIN" push $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG 
+    fi
 }
 
 cleanup() {
     rm -rf multus-cni
     git checkout -- manifests/
 }
 
+install_multus() {
```

Comment:
so this function is only invoked on kind based clusters, right ? 

Let's rename it to something that hints this is used to request a particular multus snapshot / release. 

Does that make sense to you ?

Reply:
the other way around, only on non kind

Reply:
darn. right. I hate bash.